src/entrypoints/localApp.ts

```diff
@@ -41,61 +88,49 @@ async function localApp() {
 
   const { commitHash, patchId, bundlePath, commitBranch, hasRedirects, repoDir } = await prepareBuildAndGetDependencies(
```

Comment:
Looks like with the recent changes, some of these fields are no longer used. We should consider cleaning these up to avoid any confusion in usage